### PR TITLE
CLI publishing guide for w3 (#1548)

### PR DESCRIPTION
**Pull request by ElPaisano**
Merges [Create Publish to IPFS with a CLI quickstart guide + docs review](https://github.com/ipfs/ipfs-docs/issues/1513) created by @TheDiscordian + @ElPaisano's initial edits

Finishes remaining edits, like adding the menu in

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -115,8 +116,10 @@ module.exports = {
               sidebarDepth: 1,
               collapsable: false,
               children: [
-                ['/quickstart/publish','Publish with IPFS'],
+                ['/quickstart/publish','Publish with IPFS (UI)'],
+                ['/quickstart/publish_cli', 'Publish with IPFS (command line)' ]
```
Comment (ElPaisano) [inline docs/.vuepress/config.js:120]: I ❤️ missing commas that break builds

```suggestion
                ['/quickstart/publish_cli', 'Publish with IPFS (command line)' ],
```

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -97,7 +97,8 @@ module.exports = {
...
```
Comment (ElPaisano) [inline docs/.vuepress/config.js:101]: 😭 

```suggestion
                ['/quickstart/publish_cli', 'Publish with IPFS (command line)' ], 
```

**Comment by ElPaisano** [comment]
@TheDiscordian can you give this quick pass and approve if it looks ok? 

We can ignore vale and md-link-check; in both cases, they're doing their job right, it's just that they are flagging things I havent told them to ignore. md-link-check doesn't speak anything other than http for example

https://bafybeigmokqlmeukggd5npxnbbgbsfdjah6vbxsuuulervdqqddvgnqo44.on.fleek.co/quickstart/publish_cli/

**Review by TheDiscordian — APPROVED** [review]

**Comment by 2color** [comment]
Btw, it seems the mobile view isn't ideal with the url image

**Comment by 2color** [comment]
I think it would be good to add some note about what a space is and clarifying that it isn't an IPFS specific concept.

**Comment by ElPaisano** [comment]
> Btw, it seems the mobile view isn't ideal with the url 

@2color thanks for noting this. Since the branch was merged in, I created a separate issue to investigate and fix https://github.com/ipfs/ipfs-docs/issues/1553

**Comment by ElPaisano** [comment]
> I think it would be good to add some note about what a space is and clarifying that it isn't an IPFS specific concept.

@2color sorry I'm not clear what you mean here.

**Comment by 2color** [comment]
The user is promoted to create a W3storage space. But there's clarification what it is

**Pull request merged**